SPECS/kernel/kernel.spec

```diff
@@ -327,10 +327,20 @@ then
      fi
 fi
 
+if grep -q "automatically generated by grub2-mkconfig" /boot/grub2/grub.cfg; 
```

Comment:
This will fail if we don't have mkconfig. 

Thoughts on adding a %mkconfig macro to the grub2 package that calls grub2-mkconfig, but evaluates to "" if the macro isn't present?

Reply:
I'm a bit confused about when and how this macro would be defined. Can macros be installed on a destination system and affect the behavior of already built rpms? Or are the macros expanded during RPM build time?

Reply:
Macros can affect both, but in this case since its just a Pre/Post step it needs to be present on the running system. Systemd has some good macros for post install enabling services etc. We should have a look at those. Ideally the macros would be part of the same package that provides the config folder, so we can Require(post):/Require(pre) that package and ensure the macros are there.